stdlib/public/core/Mirror.swift

```diff
@@ -289,9 +289,10 @@ public struct Mirror {
     self._makeSuperclassMirror = Mirror._superclassIterator(
       subject, ancestorRepresentation)
       
-    self.children = Children(
+    let lazyChildren =
       unlabeledChildren.lazy.map { Child(label: nil, value: $0) }
-    )
+    self.children = Children(lazyChildren)
```

Comment:
That's from a totally unrelated commit that was sitting around in my local tree. It got merged and I didn't rebase.